spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/SamplingZipkinSpanIterator.java

```diff
@@ -38,6 +41,23 @@
  * @since 1.0.0
  */
 final class SamplingZipkinSpanIterator implements Iterator<zipkin.Span> {
+	private static final List<String> ZIPKIN_CLIENT_ANNOTATIONS = Arrays.asList(
```

Comment:
this can be made a lot simpler.. just look for CLIENT_SEND. A span that has CLIENT_RECV_FRAGMENT, but not CLIENT_SEND is a bug. Also CLIENT_ADDR shouldn't be here (this is on a server span to indicate its caller)